.github/workflows/build.yml

```
@@ -558,6 +558,9 @@ jobs:
       - name: Build
         run: pnpm build:docs
 
+      - name: Test
```

```suggestion
      - name: Test docs
```

You can run the tests with `pnpm test:docs`. So we could call this step `Test docs` or `Docs tests`. The step name should explain what the step is doing.